pcsx2/Docs/readme-Docs.txt

```diff
@@ -12,14 +12,13 @@ These are not needed for building public releases.
 The legacy/ folder contains changelogs and blogs for the CVS development branch of PCSX2
 (being 0.9.4 and prior releases).
 
-IMPORTANT: Documents saved here must be of a format that everyone (or mostly everyone) can 
+IMPORTANT: Documents saved here must be of a format that everyone (or mostly everyone) can
 read! This seems to exclude Word 2007 and ODT (openoffice) formats; the latter because Word
-likes to save non-standard ODT files that OpenOffice can't read correctly (woops).  So far the 
-*best* choice seems to be the Word 97/2000 format.  It's fairly feature-full, and seems to be 
+likes to save non-standard ODT files that OpenOffice can't read correctly (woops).  So far the
+*best* choice seems to be the Word 97/2000 format.  It's fairly feature-full, and seems to be
 well-supported by most word processors these days (most importantly Word and OpenO).
 
 When editing and re-saving doc files, please make sure that you have enabled legacy
-Word 97/2000 output.  
+Word 97/2000 output.
 Word and OpenOffice should be set to default to retain the format, but may prompt you
```

Comment:
"set by default" rather than "set to default" unless we're instructing the user to do that (which I don't think is the intention here)

Reply:
I know that's not your error, but if you could correct that, while we're at it :P

Reply:
Sure :)

Reply:
Thanks xD

Reply:
What about the spelling of "Whoops"?

Reply:
Sorry about that, it has been changed. :)